### PR TITLE
Add verbosity option (incremental) and use log instead of print

### PR DESCRIPTION
**Pull request by Guts**
## Context

Recently, we faced an issue with the log length using qgis-plugin-ci on GitLab CI/CD:

```log
Job's log exceeded limit of 4194304 bytes.
Job execution will continue but no more output will be collected.
```

See this job: https://gitlab.com/Oslandia/qgis/ign-geotuileur/-/jobs/3056269109

But I've been thinking for a long time that we should reduce the amount of information returned to the terminal or let the end-user pick.

## Changes

**STILL IN PROGRESS**

- add a verbosity option `-v` with an incremental count 

   ```sh
   # by default, log nothing except print statement = WARNING
   qgis-plugin-ci package latest
   # -v = INFO
   qgis-plugin-ci -v package latest
   # -vv = DEBUG
   qgis-plugin-ci -vv package latest
   ```

- add a setting into the `setup.cfg` to customize log format
- remove option alias `-v` from the `version` option to avoid conflict
- improve some exception handling: mainly add `Trace: {exc}` and use f-string instead of format as recommended

### CODE REVIEW

File: scripts/qgis_plugin_ci.py [hunk 1]
```diff
@@ -127,6 +137,32 @@ def main():
...
+    # print the version and exit
+    if args.version:
+        import pkg_resources
+
+        print(
+            "qgis-plugin-ci version: {}".format(
+                pkg_resources.get_distribution("qgis-plugin-ci").version
+            )
+        )
+        parser.exit()
```
Comment (Gustry) [inline scripts/qgis_plugin_ci.py:164]: Left over from #153 when rebasing your PR ?

**Review by Gustry — COMMENTED** [review]
Nice move !

File: scripts/qgis_plugin_ci.py (hunk 1 above)
Comment (Guts) [reply scripts/qgis_plugin_ci.py:164]: Ups, it's all messed by a bad manip with my IDE. Sorry. I recreate a branch and a PR.

**Comment by 3nids** [comment]
niiiiice!!!

**Comment by Guts** [comment]
Thanks for your comments. I've created a clean branch and the corresponding PR: #156 . I close here.

**Pull request closed without merging**